src/main/java/com/synopsys/integration/alert/channel/slack/SlackChannel.java

```diff
@@ -87,8 +86,23 @@ public String sendMessage(final DistributionEvent event) throws IntegrationExcep
 
     public List<Request> createRequests(final DistributionEvent event) throws IntegrationException {
         final FieldAccessor fields = event.getFieldAccessor();
-        final String webhook = fields.getString(SlackDescriptor.KEY_WEBHOOK).orElseThrow(() -> new AlertException("Missing Webhook URL"));
-        final String channelName = fields.getString(SlackDescriptor.KEY_CHANNEL_NAME).orElseThrow(() -> new AlertException("Missing channel name"));
+
+        final Optional<String> webhookOptional = fields.getString(SlackDescriptor.KEY_WEBHOOK);
+        final Optional<String> channelNameOptional = fields.getString(SlackDescriptor.KEY_CHANNEL_NAME);
```

Comment:
I think these methods should return orElse(""). We then check if the string is blank or not and throw an exception. Since we are in the Channel class, we expect certain values to always be given to us. These two fields must have a value for our code to properly function otherwise we fail.